### ISSUE TITLE
Switch to systemd

### ISSUE THREAD

**Issue by babolivier** [babolivier/cozy-archlinux#7]
As most of Archlinux systems run systemd, it'd be better and wiser to use it instead of supervisor. To do this, we need to complete the following steps:
- Write and test a systemd service file
- Add its download and activation in the install process
- In the update process, remove the supervisor service `cozy-controller.ini` file (and reload supervisor's configuration)

**Comment by babolivier** [comment]
Heard that @nledez will be working on this. Don't let me down, Nicolas.

**Comment by nledez** [comment]
Hi @babolivier,

An exemple of systemd unit (/etc/systemd/system/cozy.service):
`[Unit]
Description=Cozy Controller

[Service]
Environment=NODE_ENV=production
ExecStart=/usr/bin/cozy-controller

[Install]
WantedBy=multi-user.target`

BUT if you put systemd without supervisord, the support can be painful (ping @clochix).

And add this to /etc/cozy/controller.json:
`"restart_cmd": "/bin/systemctl restart cozy"`

**Comment by babolivier** [comment]
Thanks, will look into it for future versions of the package!

**Comment by ArchangeGabriel** [comment]
@nledez @clochix Any details about this:

> BUT if you put systemd without supervisord, the support can be painful (ping @clochix).

?

**Comment by clochix** [comment]
I just opened an issue in order we document the use of systemd : https://github.com/cozy/cozy-docs/issues/327

Regarding the support, I think the main issue is that it's easier to diagnose problems if everybody use the same configuration. We may don't want to officially support both supervisord and systemd.

**Comment by ArchangeGabriel** [comment]
Oh, that kind of support. Well, regarding ArchLinux, is not like this is or will ever be an officially supported platform. ;) But we’re able to manage our own issues between us most of time. :p

And as a ArchLinux user, I would definitively go the systemd way. ;)

**Comment by nledez** [comment]
Another response is:

There are two distribution build with same package:
- Debian Jessie => With systemd
- Ubuntu Trusty => Without systemd

If we need to support them, we also need to update some scripts.

So. It's more easier for us to manage it.

Another response:

We have more priority on this Debian package before systemd.

**Comment by ArchangeGabriel** [comment]
@nledez Yes, that’s a different issue with support, but still the same meaning of supported for me: for which you (want to) provide assistance. ;)

**Comment by ArchangeGabriel** [comment]
My new package at https://aur.archlinux.org/packages/cozy/ use systemd.

The next-coming one (cozy-git, based on V3) will also do.
